hash/ibm5170.xml

```diff
@@ -11452,6 +11452,18 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="lostvik">
+		<description>The Lost Vikings</description>
+		<year>1993</year>
+		<publisher>Interplay</publisher>
+		<publisher>Silicon &amp; Synapse</publisher>
```

Comment:
Having more than one publisher tag is not allowed by the DTD

Reply:
My mistake, remove the "Silicon &amp; Synapse" the publisher tag